subcommands/waves/init.go

```diff
@@ -145,6 +150,31 @@ func doInitWave(cmd *cobra.Command, args []string) {
 	} else {
 		subcommands.DieNotNil(api.FactoryCreateWave(factory, &wave), "Failed to create a wave")
 	}
+
+	if !hasStaticDelta(new_targets) {
+
+		fmt.Print(`
+WARNING: You created a wave for a target version without static deltas.
+
+We recommend that you generate static deltas for your production targets to optimize
+OTA update download. Consider generating a static delta for targets using: 
+$ fioctl targets static-deltas
+
+You can then cancel the wave and create a new one by using a static delta.
```

Comment:
Same about semantic newlines:

```suggestion
WARNING: You created a wave for a target version without static deltas.
We recommend that you generate static deltas for your production targets to optimize OTA update download.
Consider generating a static delta for targets using:
$ fioctl targets static-deltas

You can then cancel the wave and create a new one for a target which has static deltas defined.
```

This also incorporates my above suggestion about _using static deltas_.